### PR TITLE
Modify SdsTypeCode to return a String

### PR DESCRIPTION
**Pull request by atangOSIsoft**
Asset APIs should return SdsTypeCode as Strings

### CODE REVIEW

File: Content_Portal/Documentation/Assets/assets.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ An asset or asset type metadata is static information associated with a given as
 | Id          | String | Required*  | `Id` for the metadata value.                                 |
 | Name        | String | Required  | User-friendly name for the metadata value. If not null, must be unique within an asset or asset type. |
 | Description | String | Optional  | User-provided description                                    |
-| SdsTypeCode | Int    | Required  | This integer corresponds to the SdsTypeCode. Asset metadata support the following integer values: 11 (Int64), 14 (Double), 16 (DateTime), and 18 (String). |
+| SdsTypeCode | Int    | Required  | This integer corresponds to the SdsTypeCode. Asset metadata support the following integer or string values: 11 ("Int64"), 14 ("Double"), 16 ("DateTime"), and 18 ("String"). |
```
Comment (lasato-OSI) [inline Content_Portal/Documentation/Assets/assets.md:38]:
```suggestion
| SdsTypeCode | Int    | Required  | This integer corresponds to the SdsTypeCode. Asset metadata support the following integer or string values: 11 ("Int64"), 14 ("Double"), 16 ("DateTime"), and 18 ("String"). |
```
Sorry to bring this nitpicky thing up. I guess I'm not clear on when we're using quotes.

Comment (atangOSIsoft) [reply Content_Portal/Documentation/Assets/assets.md:38]: I cannot seem to point out the differences.  I added quotes to indicate the the values in the parenthesis are string literals. Perhaps it's more clear if we separate it such as:
integer values : 11, 14, 16, or 18
corresponding string values: "Int64", "Double", "DateTime", or "String"

Comment (GiselleLeeOSI) [reply Content_Portal/Documentation/Assets/assets.md:38]: Perhaps use codeblock instead? `Int64`

Comment (lasato-OSI) [reply Content_Portal/Documentation/Assets/assets.md:38]: I think the Content Guild needs to look at this across the API doc and figure out how we're going to indicate string literals and other API elements. There is a "consistency project" and one of its goals is to address issues like this. I suggest we leave it as is and revisit this as part of this project.

**Pull request merged**